cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -545,6 +560,13 @@
     }
 }
 
+.no-js .o-mega-menu {
```

Comment:
`no-js` you mean? I think we're tied to the name because it's what modernizr swaps out.